app/controllers/properties_controller.rb

```diff
@@ -77,12 +77,10 @@ def property_children(id, acronym = params[:acronym], lang = request_lang)
   private
 
   def index_tree(container_id)
-    if !params[:propertyid].blank?
-      @root = OpenStruct.new({ children: property_tree(params[:propertyid], params[:ontology]) })
-      not_found(@root.children.errors.join) if @root.children.respond_to?(:errors)
-
+    if !params[:propertyid].blank? && (@root= OpenStruct.new({ children: property_tree(params[:propertyid], params[:ontology]) }))  && !@root.children.respond_to?(:errors)
```

Comment:
this line condition is too long and mixing the condition and the values of affection makes it hard to understand.